book/src/contributing/ast_ebnf.md

```diff
@@ -61,7 +70,7 @@ pat ::=
 | ("[" (expr ",")* "]" | "[" expr ";" int "]")
 | "&" pat
 | literal
-| <!!TODO(PBinding): ident ":=" path !!>
+| ident "@" pat
```

Comment:
The `@ pat` part should be marked optional, no?
And we should have the `mut` bit here as well